Add configurable auto-send behavior for voice recordings on release (#1362)
GetStream/stream-chat-swiftui: CHANGELOG.md

# Upcoming

### ✅ Added

> - Add `ComposerConfig.isVoiceRecordingAutoSendEnabled` to control whether releasing a recording sends it immediately or adds it to the composer preview [#1362](https://github.com/GetStream/stream-chat-swiftui/pull/1362)

Author: ```suggestion
- Add `ComposerConfig.isVoiceRecordingAutoSendEnabled` to support sending a recording instantly on release [#1362](https://github.com/GetStream/stream-chat-swiftui/pull/1362)
```
